keyboards/gck/num3x3stm32/keyboard.json

```diff
@@ -0,0 +1,140 @@
+{
+    "keyboard_name": "gck/num3x3",
+    "maintainer": "Giliy666",
+    "manufacturer":"GCK",
+    "processor": "STM32F401",
+    "development_board": "blackpill_f401",
+    "bootloader": "stm32-dfu",
+
+    "debounce":1,
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["B5", "B6", "B7"],
+        "rows": ["A3", "A4", "A5"]
+    },
+
+    "url": "www.GCK.ru",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x1313",
+        "vid": "0x1209",
+        "force_nkro":true 
+    },
+
+    "features": {
+        "command": false,
+        "console": false,   
+        "extrakey": true,
+        "mousekey": false,            
+        "nkro": true,
+        "indicators":true,      
+        "rgblight":true,
+        "rgb_matrix": false              
```

Comment:
```suggestion
        "extrakey": true,
        "mousekey": false,            
        "nkro": true,
        "rgblight":true
```

Reply:
`command` and `console` were requested for removal, and this has not been completed.